src/pages/editAccount/main.tsx

```diff
@@ -0,0 +1,92 @@
+import { useRef, useState } from 'react';
+// import Input from '../../components/input/input';
+import Input from '../../components/input/input';
+import Button from '../../components/button/button';
+import { checkValues, handler } from './hooks/hooks';
+import styles from './main.module.scss';
+import { useNavigate } from 'react-router-dom';
+
+const Changepassword = () => {
+  const [isPending, setisPending] = useState(false);
+  const oldPass = useRef<HTMLInputElement | null>(null);
+  const newPass = useRef<HTMLInputElement | null>(null);
+  const confirmPass = useRef<HTMLInputElement | null>(null);
+  const navigate = useNavigate();
+
+  async function handle() {
+    setisPending(true);
+    const divElement = document.getElementsByClassName('change-password-error')[0];
+    divElement.textContent = '';
+    const oldPassValue = oldPass.current?.value;
+    const newPassValue = newPass.current?.value;
+    const confirmPassValue = confirmPass.current?.value;
+
+    const error = checkValues(oldPassValue, newPassValue, confirmPassValue);
+    if (error) {
+      divElement.textContent = error;
+      setisPending(false);
+      return;
+    }
+    const body = {
+      oldPassword: oldPassValue,
+      newPassword: newPassValue,
+    };
+    try {
+      await handler(`${import.meta.env.VITE_BACKEND_URL}users/change-password`, body);
+      setisPending(false);
+      navigate('/');
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } catch (error: any) {
+      divElement.textContent = error.response.data.message;
+      setisPending(false);
+    }
+  }
+  return (
+    <div className={styles.cardStyle}>
+      <form action="" method="post" name="signupForm" id={styles.signupForm}>
+        <h2 className={styles.formTitle}>Change Password</h2>
+        <p>Enter your new password below</p>
+        <div className={`${styles.error} change-password-error`} data-testid="error_div">
+          {' '}
+        </div>
+        <div className={styles.inputDiv}>
+          <Input
+            textId="in"
+            placeholder="OLD PASSWORD"
```

Comment:
and here needs to be formik validations. user won't submit with fake or bad credentials